src/types.rs

```diff
@@ -75,15 +77,41 @@ impl TxLogResponse {
         let edn = parse_edn(&resp).unwrap();
         Self {
             tx___tx_id: edn[":crux.tx/tx-id"].to_string().parse::<usize>().unwrap(), 
-            tx___tx_time: edn[":crux.tx/tx-time"].to_string()
+            tx___tx_time: edn[":crux.tx/tx-time"].to_string(),
+            tx__event___tx_events: edn.get(":crux.tx.event/tx-events").map_or(None, |e| Some(e.to_string())),
```

Comment:
The `.map_or(None, |e| Some(e.to_string())` ain't the same thing as `.map(ToString::to_string)`, I don't see why to use `map_or` if the `get` will already return `None` if it doesn't exists.

Reply:
0db3d94